<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>edit</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="table_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QTextEdit" name="edit_1">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit_2">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
